docs/guides/apache-sites-enabled.md

```diff
@@ -79,11 +78,11 @@ The configuration file configuration content would look something like this:
 ```
 Once the file is created, we need to write (save) it with: `shift : wq`
 
-In our example above, the wiki site is loaded from the html sub-directory of _com.ourownwiki.www_, which means that the path we created in _/var/www_ (above) will need some additional directories to satisfy this:
+In our example above, the wiki site loads from the HTML sub-directory of _com.ourownwiki.www_, which means that the path we created in _/var/www_ (above) will need some additional directories to satisfy this:
 
```

Comment:
No. It actually loads according to the configuration file from the "html" sub-directory.

Reply:
Yes but the paragraph I was correcting did not say that. I just corrected grammar here and nothing is changed with the correction; only the meaning is enhanced.
HTML should be in upper case.

Reply:
'is loaded' is grammatically incorrect since it is the passive voice.

Reply:
Probably "It actually loads according to the configuration file from the "html" sub-directory" should be included in this section to actually explain how the process takes place. That would have been for my second pass. Initially I intend to proof the entire Document Folder for Grammatical and Semantic erros and the go back and cover what should be added additionally. I also have several tutorials to add if people would like to see them.

I worked in Academia for over 20 years, Computer Engineering and Electrical Engineering (amongst many other disciplines), was director of the Information technology Department at one University, and one thing I found with Academics and it is more true today than ever; is that they personally identify with their writings and their academic work.

The friction occured when their work was critcized and they then took it as a personal attack. I do not have those issues. If someone criticizes my work, I look at the critcism and see if it will make my work better and then incorporate it in my work. My work ends up being everyone's work and that is why I use the Creative Commons CCO license for Intellectual property, see:
https://creativecommons.org/share-your-work/public-domain/cc0/
So the issue now to determine is where the corrections submitted change the meaning of what was there before, do those changes make the statement more or less understandable, and how can a system progress if it does not get better. My only intent here is to help the Documentation  evolve in a more understandable direction. I could care less about who gets the credit or whose egos may be stroked.
Best wishes

Gerry

Garthus 27 June 2021.

Reply:
When we are referring to html as a sub-directory, not the protocol, at least in this case the directory is lower case. Just follow the example and it shows you. So the HTML protocol, but in this case the "html" sub-directory. 

I'm fine with making changes where needed. Documents are never (or should never) be static. Again, I don't want the document changed to break the function. If it can be made more readable, more accurate grammatically, then by all means change it. If something is broken technically, that should be changed as well, but that isn't likely with the documents I've written, because they were written while lab testing the steps.

I have no experience in academia. Again, no English teacher here. My experience is 21 years of hard-knocks System Administration and Network Administration, in a 99% Linux only environment. I began volunteering for Rocky Linux just before my retirement in January to keep my mind sharp technically. It's mostly been a good experience, and I hope that your experience will also be a good one.

You will probably have to push your changes again for this PR, since there have been a lot of changes since it was first created. Just (please) make sure that the meanings in the procedure don't change because of your proposed edits.

Thanks,
Steve

sspencerwire